guides/source/configuring.md

```diff
@@ -1131,51 +1131,13 @@ config.relative_url_root = "/app1"
 alternatively you can set the `RAILS_RELATIVE_URL_ROOT` environment
 variable.
 
-Rails will now prepend "/app1" when generating links.
-
 #### Using Passenger
 
 Passenger makes it easy to run your application in a subdirectory. You can find the relevant configuration in the [Passenger manual](https://www.phusionpassenger.com/library/deploy/apache/deploy/ruby/#deploying-an-app-to-a-sub-uri-or-subdirectory).
 
 #### Using a Reverse Proxy
 
-Deploying your application using a reverse proxy has definite advantages over traditional deploys. They allow you to have more control over your server by layering the components required by your application.
```

Comment:
I'm not sure I agree with this removal. The NGINX example is related to subdirectory deployment because it can appear to be hosted on a subdirectory when contacting the proxy server. If we can link to a place with this information with clearer examples ([maybe here?](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)), that would be ok but we're being less helpful by removing this altogether.

Reply:
Sorry, I don't understand. 
> because it can appear to be hosted on a subdirectory when contacting the proxy server
* What do you mean by "it"? 
* Who's contacting the proxy server? Shouldn't that be the browser?

Have to admit that I'm not a NGINX expert, but subdirectory `/app1` from the example _is not appearing at all_ in the "Using a Reverse Proxy" section. So for me I cannot see the difference to a "standard" reverse proxy setup. 

Having some documentation about "reverse proxy" deployment is also good, but it's in the wrong place here. If there was a "Deployment" guide, I'd move it there, but since there's no page like this, it would be better to remove it.

Reply:
The "Using a Reverse Proxy" section has some more issues:
* Words "Reverse proxy server", "proxy server", "web server" are all used for the same thing
* "... to accept connections from your application server (Unicorn) ... " - it should rather be " ... to accept connections **for** your application server"
* Unicorn is not the best example anymore since Puma and Passenger are more popular now 
* Passenger can be run behind a reverse proxy too, so it's a little confusing why it's mentioned in a separate section
* The first two paragraphs are only understandable if you already know what a reverse proxy is. And in that case, they don't tell you anything new.

That's also why I suggest to remove it.

Reply:
The docs here can definitely be improved, but I'm just hesitant to remove the section without providing something in its place. I think we should opt to rewrite instead of remove.

I mainly meant that if you have a reverse proxy delegating to several app servers at different `location`s, I believe each app would need to have `config.relative_url_root` configured to display properly when the browser accesses them via the proxy, right?

Reply:
Here's what I think might be a better change:

`
In this case, you would need to configure the proxy server to forward connections to your application server (Puma, Unicorn, etc). Check your proxy server's documentation ([NGINX](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/), [Apache](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html)) for instructions on how to map application servers to paths.
`